src/pybind/mgr/dashboard/module.py

```diff
@@ -65,12 +65,24 @@ def recurse_refs(root, path):
 def get_prefixed_url(url):
     return global_instance().url_prefix + url
 
+def prepare_url_prefix(url_prefix):
+    if url_prefix == None:
+            url_prefix = ''
```

Comment:
probably we can use #19948 to simplify this a little bit?

Reply:
I'll change this as soon as #19948 is merged.